sync/src/relayer/compact_block_process.rs

```diff
@@ -226,8 +226,6 @@ impl<'a> CompactBlockProcess<'a> {
                 }
             }
 
-            assert!(!missing_transactions.is_empty() || !missing_uncles.is_empty());
```

Comment:
What happens if code runs here and both `missing_transactions` and `missing_uncles` are empty?